draft-ietf-rtcweb-jsep.xml

```diff
@@ -3199,12 +3212,20 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               the size limits specified in
               <xref target="RFC5245" />, Section 15.4.</t>
 
+              <t>dtls-id value, which MUST be set according to
+              <xref target="I-D.ietf-mmusic-dtls-sdp"/> Section 5.
+              If this is a re-offer and the dtls-id value is different
+              from that presently in use, then a new DTLS connection
+              MUST be established, and the ufrag and password values
+              MUST also be different. If processing an answer, the
```

Comment:
I would spell this out, i.e., if we're doing a new DTLS connection here, it MUST be as part of a proposed ICE restart, with new ICE ufrag and password values.